opentracing-web-servlet-filter/src/main/java/io/opentracing/contrib/web/servlet/filter/ServletFilterSpanDecorator.java

```diff
@@ -137,5 +141,9 @@ public void onTimeout(HttpServletRequest httpServletRequest, HttpServletResponse
 
             return errorLog;
         }
+
```

Comment:
Anything above HTTP status 400 is considered error

Reply:
This will probably brake other integration building on top of this component.